ush/config_defaults.yaml

```diff
@@ -5,7 +5,6 @@ metadata:
   description: >-
     Default configuration for an experiment. The valid values for most of the
     parameters are specified in valid_param_vals.yaml
-  version: !!str '1.0'
```

Comment:
It isn't used, as far as I can tell. If it were, it would need to change with every change of this file. It would likely be best to just track the hash of the repo that was used for an experiment. That would be the only thing that would really come close to ensuring compatibility of config files, although I'm not sure even that would be a strong 100% guarantee.

Until there is some sort of automated method for updating this number, it's fallible.